ddtrace/bootstrap/sitecustomize.py

```diff
@@ -84,12 +86,12 @@ def update_patched_modules():
     if asbool(get_env("runtime_metrics", "enabled")):
         RuntimeWorker.enable()
 
-    opts = {}
+    opts = {}  # type: Dict[str, Any]
 
     if asbool(os.environ.get("DATADOG_TRACE_ENABLED", True)):
-        patch = True
+        trace_enabled = True
```

Comment:
Renamed this variable as mypy confused this with the `patch()` function from monkey.py that was imported into sitecustomize.

Reply:
👍🏻